src/index.js

```diff
@@ -33,6 +33,18 @@ const SanitizedHTML = props => {
     sanitizerOptions
   );
 
+  if(props.tagName){
```

Comment:
I guess, you forgot to set tagName types in PropTypes

Reply:
Yep, looks like it!